src/Commands/TerminusCommand.php

```diff
@@ -41,4 +42,15 @@ protected function log()
     {
         return $this->logger;
     }
+
+    /**
+     * Override Robo's IO function with our custom style.
+     */
+    protected function io()
+    {
+        if (!$this->io) {
+            $this->io = new TerminusStyle($this->input(), $this->output());
+        }
+        return $this->io;
+    }
```

Comment:
Either way.  This way, it's lazy-evaluated, whereas the other way it isn't. I don't think that's super-important to lazy-evaluate here, though.